src/app/studies/containers/studies-page.html

```diff
@@ -1,35 +1,46 @@
 <app-toolbar>
   <div start>
-    <img class="logo" src="assets/radar-logo.svg" alt="RADAR-base Logo">
+    <img
+      class="logo"
+      src="assets/radar-logo.svg"
+      alt="RADAR-base Logo"
+    >
     <div class="label">
-      <div class="font-body-light">RADAR-base</div>
-      <div class="font-title">Studies</div>
+      <div class="font-body-light">RADAR-CNS Dashboard</div>
+      <div class="font-title">Mental Health</div>
     </div>
   </div>
 </app-toolbar>
 
-<div
-  class="container"
-  *ngIf="studiesIsLoaded$ | async"
+<app-tile
+  label="Studies Summary"
+  class="tile-studies"
 >
-  <div class="grid-wrapper">
-    <div class="grid">
-      <div class="study-card"
-        *ngFor="let study of studies$ | async"
-        (click)="openStudy(study.projectName)">
-        <h2 class="title">
-            {{ study.projectName }}
-        </h2>
-        <h4 class="subtitle">
-          <span>{{ study.id }}</span>
-          <span *ngIf="study.location"> | {{ study.location }}</span>
-          <span *ngIf="study.organization"> | {{ study.organization }}</span>
-          <span *ngIf="study.projectStatus"> | {{ study.projectStatus }}</span>
-        </h4>
-        <div class="description">
-          {{ study.description }}
+  <div tile-content>
+
+    <div
+      class="container"
+      *ngIf="studiesIsLoaded$ | async"
+    >
+      <div class="grid-wrapper">
+        <div class="grid">
+          <div
+            class="study-card"
+            *ngFor="let study of studies$ | async"
+            (click)="openStudy(study.projectName)"
+          >
+            <h2 class="title">{{ study.projectName }}</h2>
+            <h4 class="subtitle">
+              <span>{{ study.id }}</span>
+              <span *ngIf="study.location">| {{ study.location }}</span>
```

Comment:
can you please add the space back before the `|`? `<span *ngIf="study.location"> | {{ study.location }}</span>`